pkg/acsengine/const.go

```diff
@@ -17,6 +17,8 @@ const (
 	DefaultKubernetesMasterSubnet = "10.240.0.0/16"
 	// DefaultKubernetesClusterSubnet specifies the default subnet for pods.
 	DefaultKubernetesClusterSubnet = "10.244.0.0/16"
+	// DefaultDockerBridgeSubnet specifies the default subnet for the docker bridge network for masters and agents.
+	DefaultDockerBridgeSubnet = "172.17.0.5/16"
```

Comment:
Based on the docs from here: https://docs.docker.com/engine/userguide/networking/default_network/custom-docker0/

I am using the --bip flag, which is used to set a static ip + subnet mask in one shot.

--bip=CIDR: supply a specific IP address and netmask for the docker0 bridge, using standard CIDR notation. For example: 192.168.1.5/24